app/components/administrative-body-select.js

```diff
@@ -0,0 +1,53 @@
+import { inject as service } from '@ember/service';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { task } from 'ember-concurrency';
+
+const VALID_ADMINISTRATIVE_BODY_CLASSIFICATIONS = [
```

Comment:
I see you've hardcoded this now, not sure I'm a fan of that. I have to admit this is not data that changes often.

Reply:
nm, I see it was already hardcoded. oh well :)

Reply:
Yea I just moved it from the other component and defined it as a const instead. If you give me pointers on how to do it without hardcoding I can take a look at it if you want :smiley:.